indico/modules/events/registration/templates/management/_sortable_list.html

```diff
@@ -0,0 +1,28 @@
+{% macro sortable_items(items, actions_macro) -%}
+    {% for item in items %}
+        <li class="i-label sortable-item" id="{{ item.id }}">
```

Comment:
not a big fan of using the `id` like this. I'd rather not set an actual ID and use `data-id` or `data-item-id` instead.